libc/src/__support/CPP/new.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "new.h"
+#include <stdlib.h>
```

Comment:
Consider adding a comment on what's used (`free`).

Reply:
ok will do that right now.